ajax/libs/shaka-player/package.json

```diff
@@ -14,7 +14,8 @@
     {
       "basePath": "dist",
       "files": [
-        "shaka-player*"
+        "shaka-player*",
+        "controls.css*"
```

Comment:
Are `demo.*` or `receiver.*` files needed? Or just `shaka-player*` and `controls.css*`?

```
ls node_modules/shaka-player/dist

controls.css                           demo.compiled.map                      receiver.compiled.debug.js             shaka-player.compiled.debug.js         shaka-player.ui.debug.externs.js       shaka-player.ui.map
controls.css.map                       demo.css                               receiver.compiled.debug.map            shaka-player.compiled.debug.map        shaka-player.ui.debug.js               wrapper.js
demo.compiled.debug.js                 demo.css.map                           receiver.compiled.js                   shaka-player.compiled.externs.js       shaka-player.ui.debug.map
demo.compiled.debug.map                deps.js                                receiver.compiled.map                  shaka-player.compiled.js               shaka-player.ui.externs.js
demo.compiled.js                       locales.js                             shaka-player.compiled.debug.externs.js shaka-player.compiled.map              shaka-player.ui.js
```

Reply:
Nope.  Those are part of our demo app, and shouldn't be needed by other people's apps.